client/src/components/small-display/SmallDisplayCreateJoinModal.vue

```diff
@@ -0,0 +1,142 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-page id="create-join-modal">
+    <create-organization-modal-header
+      @close-clicked="onCloseRequested"
+      title="HomePage.noExistingOrganization.createOrganization"
+    />
+    <div class="create-join-modal-content">
+      <ion-text class="create-join-modal-content__description">
+        {{ $msTranslate('HomePage.noExistingOrganization.createOrganizationDescription') }}
+      </ion-text>
+      <ion-list class="create-join-modal-list">
+        <ion-item
+          class="create-join-modal-list__item"
+          @click="clicked(HomePageAction.CreateOrganization)"
+        >
+          <ion-icon
+            :icon="addCircle"
+            slot="start"
+            class="item-icon"
+          />
+          <div class="item-content">
+            <ion-label class="item-content__title subtitles-normal">
+              {{ $msTranslate('HomePage.noExistingOrganization.createOrganizationTitle') }}
+            </ion-label>
+            <ion-text class="item-content__description body">
+              {{ $msTranslate('HomePage.noExistingOrganization.createOrganizationSubtitle') }}
+            </ion-text>
+          </div>
+        </ion-item>
+        <ion-item
+          class="create-join-modal-list__item"
+          @click="clicked(HomePageAction.JoinOrganization)"
+        >
+          <ion-icon
+            :icon="mailUnread"
+            slot="start"
+            class="item-icon"
+          />
+          <div class="item-content">
+            <ion-label class="item-content__title subtitles-normal">
+              {{ $msTranslate('HomePage.noExistingOrganization.joinOrganizationTitle') }}
+            </ion-label>
+            <ion-text class="item-content__description body">
+              {{ $msTranslate('HomePage.noExistingOrganization.joinOrganizationSubtitle') }}
+            </ion-text>
+          </div>
+        </ion-item>
+      </ion-list>
+    </div>
+  </ion-page>
+</template>
+
+<script setup lang="ts">
+import { IonPage, IonIcon, IonItem, IonLabel, IonList, IonText, modalController } from '@ionic/vue';
+import { addCircle, mailUnread } from 'ionicons/icons';
+import { HomePageAction } from '@/views/home/HomePageButtons.vue';
+import { MsModalResult } from 'megashark-lib';
+import CreateOrganizationModalHeader from '@/components/organizations/CreateOrganizationModalHeader.vue';
+
+const emits = defineEmits<{
+  (e: 'click', action: HomePageAction): void;
+  (e: 'closeRequested'): void;
+}>();
+
+const props = defineProps<{
+  replaceEmit?: (action: HomePageAction) => Promise<void>;
+}>();
+
+async function clicked(action: HomePageAction): Promise<void> {
+  if (props.replaceEmit) {
+    await props.replaceEmit(action);
```

Comment:
Why did you opt for this `replaceEmit` prop? It would be simpler to call `return await modalController.dismiss({ action: action });` and replace this `replaceEmit` prop with a boolean indicating if you need to return something or not. But I'm not sure that's even needed.